packages/ts-docs-gen/src/abstractions/container-plugin.ts

```diff
@@ -51,6 +51,18 @@ export abstract class ContainerPlugin<TKind extends ApiContainer> extends BasePl
         return result;
     }
 
+    // -----
+
+    // protected abstract ContainerMembersKindsGroups
+
+    protected HeadingMembers: PluginHeading[] = [];

```

Comment:
For what are you using `property` in plugin? I think render method should only be used like in functional programming.